submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/service/ParentInfoService.kt

```diff
@@ -11,9 +13,13 @@ class ParentInfoService(private val queryService: SubmissionQueryService) {
     }
 
     private fun parentInfo(parentAccNo: String): ParentInfo {
-        require(queryService.existByAccNo(parentAccNo)) { "The project '$parentAccNo' doesn't exist" }
+        require(queryService.existByAccNo(parentAccNo)) { throw ProjectNotFoundException(parentAccNo) }
+
+        val accessTags = queryService.getAccessTags(parentAccNo).filterNot { it == SubFields.PUBLIC_ACCESS_TAG.value }
```

Comment:
I am wondering if we should query-specific project access tag which match accNo instead of getting all of them by relation.

Reply:
I get your point but we would still need to get all the other tags for the parent submission so it would be two database calls instead of one

Reply:
agree